src/schema/notifications.ts

```diff
@@ -98,13 +119,18 @@ export class ZimbraNotifications {
 				const id = findDataId(this.cache, '$ROOT_QUERY.search', dataId =>
 					r.test(dataId)
 				);
+				const sortByValue = getVariablesFromDataId(id)
+					? getVariablesFromDataId(id).sortBy
```

Comment:
const { sortBy } = getVariablesFromDataId(id)

Reply:
while deleting contact groups from we get undefined id so need to set those conditions